### ISSUE TITLE
npm-groovy-lint fails due to the error "Unable to start CodeNarc server" which may require jdk/jre 17

### ISSUE THREAD

**Issue by monicathx** [oxsecurity/megalinter#3175]
**Describe the bug**
I've downloaded latest version **v7.6.0** at my local, and build the docker image of Megalinter and run the image at local to lint one of my local repository in which there is Jenkins file recoganized as groovy script to be linted . 

The "Groovy" linter(npm-groovy-lint) fails with errors as below from the Megalinter console report:

```html
Results of npm-groovy-lint linter (version 13.0.0)
See documentation on https://megalinter.io/latest/descriptors/groovy_npm_groovy_lint/
-----------------------------------------------

❌ [ERROR] for workspace /tmp/lint
Linter raw log:
Java jre or jdk 17 is required
Installing Java jre 17 in /root/.java-caller...
Installed Java jre 17 in /root/.java-caller/jre...
Unable to start CodeNarc server: {"status":1,"stdout":"","stderr":"","childJavaProcess":{"_events":{},"_eventsCount":2,"_closesNeeded":1,"_closesGot":1,"connected":false,"signalCode":null,"exitCode":1,"killed":false,"spawnfile":"java","_handle":null,"spawnargs":["java","-Xms256m","-Xmx2048m","-jar","/node-deps/node_modules/npm-groovy-lint/lib/java/CodeNarcServer.jar","--server"],"pid":340,"stdin":null,"stdout":null,"stderr":null,"stdio":[null,null,null]}}
["--server"]
GroovyLint: Error running CodeNarc: 
``` 

I suppose the root cause should be the `Openjdk` version installed in Megalinter is not compatible with the latest version of `npm-groovy-lint` , it shows in the Megalinter report that this linter is already the latest 13.0.0, and according to their github project [releases information](https://github.com/nvuillam/npm-groovy-lint/releases), it requires jdk17 otherwise it will fail the process. But in v7.6.0 it is still openjdk11, see the code: https://github.com/oxsecurity/megalinter/blob/5364b811bbe86576d832227be174ebc2aa0f5f49/megalinter/descriptors/groovy.megalinter-descriptor.yml#L41 

However, I see in the `main` branch (https://github.com/oxsecurity/megalinter/blob/9d444629ce436925fd66c7e2ef966bf5562351b4/megalinter/descriptors/groovy.megalinter-descriptor.yml#L41), this issue has been fixed, so I am wondering if you are planning to publish a new release including this issue? 

Anyway, this issue is easy to fix, either change the version of openjdk back to 11, or specify the older version of `npm-groovy-lint` as: 
      ```npm-groovy-lint@11.1.1```

**To Reproduce**
Download latest version **v7.6.0** at my local, 
build the docker image of Megalinter and run the image at local to lint the repository in which there is Jenkins file recoganized as groovy script to be linted . 

**Expected behavior**
It should not throw this kind of error which is not related the script itself

**Screenshots**
No

**Comment by echoix** [comment]
Is this still an issue?

**Comment by nvuillam** [comment]
I'm pretty sure it is solved in [7.7.0 ](https://github.com/oxsecurity/megalinter/releases) :)

[npm-groovy-lint](https://nvuillam.github.io/npm-groovy-lint/) from 12.1.0 to 13.0.2
